<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRC</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CRC</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_send">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>391</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字节序列：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>264</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>协议：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_parser">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&gt;&gt; Parser &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_program">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>char 0x5A 	// HEAD1
char 0xA5		// HEAD2	
char 1		// SRC
char 2		// DST
char 10		// TYPE
char 0		// SN
char  3		// ID
char 5		// SUBID
short 22  	// LEN
char 0x66		// DATA
char 'x'		// DATA
int 3		// DATA
float 3.5	 	// DATA
double 7.6	// DATA
str &quot;cute&quot;	// DATA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
